packages/popover/src/positionUtils.ts

```diff
@@ -125,13 +164,14 @@ interface TransformOriginArgs {
 type XOrigin = 'left' | 'right' | 'center';
 type YOrigin = 'top' | 'bottom' | 'center';
 
-const yJustifyOrigins: { [J in Justify]: YOrigin } = {
+const yJustifyOrigins: Record<Justify, YOrigin> = {
```

Comment:
These aren't equivalent! Using `Record` means it won't fail if we add a value to the `Justify` enum in the future.

Reply:
Interesting – I just tested this, and adding a new property to the Justify enum does cause an error because the property is missing.

Reply:
Looking into it, the definition for the Record helper is:
```ts
/**
 * Construct a type with a set of properties K of type T
 */
type Record<K extends keyof any, T> = {
    [P in K]: T;
};
```

Reply:
Oh, I stand corrected, then!